frontend/app/tab/tab.scss

```
@@ -96,6 +120,24 @@
         transition: none !important;
     }
 
+    &.has-indicator {
+        .tab-inner {
+            background: rgb(from var(--tab-indicator-color) r g b / 0.15);
```

Stylelint (17.2.0)

[error] Expected "0.15" to be "15%" (alpha-value-notation)

(alpha-value-notation)